clang/test/dpct/cooperative_groups.cu

```diff
@@ -1,7 +1,8 @@
 // UNSUPPORTED: cuda-8.0
 // UNSUPPORTED: v8.0
-// RUN: dpct --format-range=none -out-root %T/cooperative_groups %s --cuda-include-path="%cuda-path/include" --use-experimental-features=nd_range_barrier,logical-group -- -x cuda --cuda-host-only -std=c++14
+// RUN: dpct --format-range=none -out-root %T/cooperative_groups %s --cuda-include-path="%cuda-path/include" --use-experimental-features=root-group,logical-group -- -x cuda --cuda-host-only -std=c++14
```

Comment:
keep some test for option --use-experimental-features=nd_range_barrier, as it is not removed.

Reply:
Add the cooperative_groups_nd_range_barrier.cu test case